ansible/deploy-frontend.yml

```
@@ -0,0 +1,162 @@
+      become_user: www-data
+      environment:
+        NODE_OPTIONS: "--max-old-space-size=2048"
+
+    - name: Create nginx configuration
+      copy:
+        dest: /etc/nginx/sites-available/default
+        content: |
+          server {
+              listen 80;
+              server_name _;
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)